extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcRecorder.java

```diff
@@ -171,6 +171,14 @@ public void handle(AsyncResult<OAuth2Auth> event) {
                 });
 
                 auth = cf.join();
+
+                String endSessionEndpoint = OAuth2AuthProviderImpl.class.cast(auth).getConfig().getLogoutPath();
+
+                if (endSessionEndpoint == null && oidcConfig.logout.path.isPresent()) {
+                    throw new RuntimeException(
```

Comment:
`OIDCException`

Reply:
But if I throw a `OIDCException` it is going to retry the discovery, right ? I just want to fail fast.

Reply:
Yeah, that is fine